subsys/bluetooth/controller/ll_sw/ull_scan_aux.c

```diff
@@ -562,35 +561,10 @@ void ull_scan_aux_setup(memq_link_t *link, struct node_rx_hdr *rx)
 
 void ull_scan_aux_done(struct node_rx_event_done *done)
 {
-	struct ll_scan_aux_set *aux;
-	struct ull_hdr *hdr;
-
-	/* Get reference to ULL context */
-	aux = CONTAINER_OF(done->param, struct ll_scan_aux_set, ull);
-
-	if (0) {
-#if defined(CONFIG_BT_CTLR_SYNC_PERIODIC)
-	} else if (!ull_scan_aux_is_valid_get(aux)) {
-		struct ll_sync_set *sync;
-
-		sync = CONTAINER_OF(done->param, struct ll_sync_set, ull);
-		LL_ASSERT(ull_sync_is_valid_get(sync));
-		hdr = &sync->ull;
-
-		if (!sync->lll.lll_aux) {
-			return;
-		}
-
-		aux = HDR_LLL2ULL(sync->lll.lll_aux);
-#endif /* CONFIG_BT_CTLR_SYNC_PERIODIC */
-	} else {
-		/* Setup the disabled callback to flush the auxiliary PDUs */
-		hdr = &aux->ull;
-	}
-
-	LL_ASSERT(!hdr->disabled_cb);
-	hdr->disabled_param = aux;
-	hdr->disabled_cb = done_disabled_cb;
+	/* No need to do anything, aux context is flushed either from PDU node
+	 * or aux release node.
+	 */
+	/* TODO: remove? */
```

Comment:
`ull_scan_aux_done` is needed when Periodic Advertising is terminated or pre-empted by other state/role, in which case `isr_rx` gets overrided by `isr_done` when `abort_cb` is called.